hack/ginkgo-e2e.sh

```diff
@@ -136,7 +136,8 @@ fi
 export PATH=$(dirname "${e2e_test}"):"${PATH}"
 "${ginkgo}" "${ginkgo_args[@]:+${ginkgo_args[@]}}" "${e2e_test}" -- \
   "${auth_config[@]:+${auth_config[@]}}" \
-  --ginkgo.flakeAttempts="${FLAKE_ATTEMPTS}" \
+  --ginkgo.focus="should orphan pods created by rc if delete options say so" \
+  --ginkgo.flakeAttempts="100" \
```

Comment:
@krzyzacy i was trying to run this tests 100 times. According to the ginkgo doc, this flag is going to do the trick, but it doesn't. Do you happen to know why?